examples/light-switch-app/efr32/include/AppConfig.h

```diff
@@ -21,6 +21,9 @@
 
 // ---- Lighting Example App Config ----
 
+// Software Version
+#define SW_VERSION "1.0.0"
```

Comment:
We have a software version #define for all apps already. We should reuse that

`#define CHIP_DEVICE_CONFIG_DEVICE_SOFTWARE_VERSION_STRING "0.1ALPHA"`